admin/installer/docker-swarm-ca-setup.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Generate certs for {Docker, Swarm} TLS and start Swarm Manager.
+set -ex
+
+: ${s3_bucket:?}
+
+# Get expect to autofill openssl inputs
+sudo apt-get install -y expect
+
+PASSPHRASE=$(dd bs=18 count=1 if=/dev/urandom | base64 | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+
+DOCKER_CERT_HOME='/root/.docker'
+mkdir -p ${DOCKER_CERT_HOME}
+cat > ${DOCKER_CERT_HOME}/passphrase.txt << EOF
+${PASSPHRASE}
+EOF
+
+# Generate CA private and public keys
+cat > ${DOCKER_CERT_HOME}/createca1.exp << EOF
+#!/usr/bin/expect -f
+set timeout -1
+spawn openssl genrsa -passout stdin -aes256 -out ${DOCKER_CERT_HOME}/ca-key.pem 4096
+match_max 100000
+send -- "${PASSPHRASE}\r"
+expect eof
+EOF
+chmod +x ${DOCKER_CERT_HOME}/createca1.exp
+${DOCKER_CERT_HOME}/createca1.exp
+
+cat > ${DOCKER_CERT_HOME}/createca2.exp << EOF
+#!/usr/bin/expect -f
+set timeout -1
+spawn openssl req -new -x509 -days 365 -key ${DOCKER_CERT_HOME}/ca-key.pem -sha256 -out ${DOCKER_CERT_HOME}/ca.pem
```

Comment:
:grey_question: @wallrj Seemed to think there might be a config file for openssl that we could use instead of using expect. Contemplate using that instead. I feel like that might be less brittle.